src/client/components/All-meals.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from "react";
+import Header from "./Header";
+import Footer from "./Footer";
+import DataContext from "./DataContext";
+import { Link } from "react-router-dom";
+
+export default function Meals() {
+  const { meals, loading, error } = useContext(DataContext);
+
+  if (loading) return <h3>Loading...</h3>;
+  if (error) return <p>{error}</p>;
+
+  return (
+    <div>
+      <Header />
+      <div className="centerAlign">
+        <ol>
+          {!meals || meals.length === 0 ? (
+            <li>No Meals</li>
+          ) : (
+            meals.map((meal) => {
+              return (
+                <div className="mealBorder">
+                  <li key={meal.id} className="boldFont">
+                    {meal.title}{" "}
+                  </li>
+                  <li>{meal.description}</li>
+                  <li>{meal.price} DKK</li>
+                  <li>Event on: {meal.when_date.slice(0, 10)}</li>
+                </div>
+              );
+            })
+          )}
+        </ol>
```

Comment:
This looks like a component that could live on their own, "MealsList"?

The React philosophy is really to create very, very small components so don't be scared to extract things!